[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\shery\\AppData\\Local\\Temp\\quarto-session125ca0b3f05f8819'\n"]
